blog/templates/blog/admin/post_list.html

```
@@ -0,0 +1,635 @@
+                href="?search={{ search }}&category={{ category_id }}&is_pinned={{ is_pinned }}&ordering={{ ordering }}&page={{ page_obj.previous_page_number }}#post-list-section"
+            >
+                <img src="{% static 'blog/image/previewicon.svg' %}" alt="이전">
+            </a>
+            {% else %}
+            <span class="page-btn page-arrow disabled">
+                <img src="{% static 'blog/image/previewicon.svg' %}" alt="이전">
+            </span>
+            {% endif %}
+
+            {% for num in page_obj.paginator.page_range %}
+                {% if num >= page_obj.number|add:-2 and num <= page_obj.number|add:2 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)